src/material-experimental/mdc-dialog/testing/dialog-harness.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {HarnessPredicate} from '@angular/cdk/testing';
+import {
+  DialogHarnessFilters,
+  MatDialogHarness as NonMdcDialogHarness
+} from '@angular/material/dialog/testing';
+
+/** Harness for interacting with a standard `MatDialog` in tests. */
+export class MatDialogHarness extends NonMdcDialogHarness {
```

Comment:
Not something to do in this PR, but it seems like this harness is a little lacking in methods...

Reply:
Can you elaborate? It currently provides the same method as the non-MDC one, or do you think we miss some methods there too?

Reply:
Oh I missed the fact that this was inheriting some methods from the non-mdc one. Though still, looking at the non-MDC one, there's no way to get a `HarnessLoader` for the content which seems pretty important. It might also make sense to add a method to get the title and get a loader for the actions area

Reply:
That's fair. How about we tackle that in a separate PR?

Reply:
Yeah like I said, not something to do in this PR

Reply:
👍

Reply:
Created COMP-312 for tracking.